controller/src/ttl_store/ttl_map.rs

```diff
@@ -26,48 +26,46 @@ impl<K: Hash + Eq + Clone, V> TtlMap<K, V> {
             ttl,
             inner_map: HashMap::new(),
             queue: VecDeque::new(),
-            last_time: DateTime::<Utc>::MIN_UTC,
+            last_time: SystemTime::UNIX_EPOCH,
```

Comment:
what's this for? is `last_time` supposed to be the last_time of access? its not updated anywhere though.

Reply:
Ah good catch, it's supposed to be the time of the last compaction, but I wasn't setting it. I've renamed it `last_compaction` and ensured that it is set.